exist-versioning-release.md

```diff
@@ -150,48 +231,72 @@ You will require a system with:
             <!-- eXist-db Github Release -->
             <server>
                 <id>github</id>
-                <privateKey>[Github Personal access tokens]</privateKey>
+                <privateKey>YOUR-Github-ACCESS-TOKEN</privateKey>
             </server>
         </servers>
+        
+        <profiles>
+           <profile>
+               <id>existdb-release-signing</id>
+               <properties>                   
+                      
+                   <existdb.release.key>YOUR-GPG-KEY-ID</existdb.release.key>
+                   <existdb.release.public-keyfile>/your/part/to/.gnupg/pubring.gpg</existdb.release.public-keyfile>
+                   <existdb.release.private-keyfile>/your/part/to/.gnupg/secring.gpg</existdb.release.private-keyfile>
+                   <existdb.release.key.passphrase>YOUR-GPG-PASSWORD</existdb.release.key.passphrase>
+
+                   <existdb.release.keystore>/your/path/to/.keystore</existdb.release.keystore>
+                   <existdb.release.keystore.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.pass>
+                   <existdb.release.keystore.key.alias>YOUR-KEYSTORE-ALIAS</existdb.release.keystore.key.alias>
+                   <existdb.release.keystore.key.pass>YOUR-KEYSTORE-PASSWORD</existdb.release.keystore.key.pass>
```

Comment:
`YOUR-KEYSTORE-PASSWORD` -> `YOUR-KEY-PASSWORD`

Reply:
These are not necessarily the same. They are the same **if** you are using PKCS12 format, but not all codesigning certificates use that format, legacy keystore for example uses jks format.

Reply:
Would it be enough to state at the beginning only PKCS12 keystores are considered in this guide?